### ISSUE TITLE
volmap segmentation fault

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#424]
catch you. :P 

script
```bash
parm data/tz2.ortho.parm7
trajin data/tz2.ortho.nc 1 1
rms first :1-13
center :1-13 mass origin
volmap output/dummy.dat 0.5 0.5 0.5 :WAT@O  centermask !:1-13 radscale 1.36 size 20,20,20 
```

I have this test in pytraj and only get segmentation fault after you added openmp for `volmap`.

log
```bash

CPPTRAJ: Trajectory Analysis. V17.00b OpenMP
    ___  ___  ___  ___
     | \/ | \/ | \/ | 
    _|_/\_|_/\_|_/\_|_

| Date/time: 11/23/16 17:30:18
| Available memory: 639.664 MB

INPUT: Reading input from 't.in'
  [parm data/tz2.ortho.parm7]
	Reading 'data/tz2.ortho.parm7' as Amber Topology
	Radius Set: modified Bondi radii (mbondi)
	This Amber topology does not include atomic numbers.
	Assigning elements from atom masses/names.
  [trajin data/tz2.ortho.nc 1 1]
	Reading 'data/tz2.ortho.nc' as Amber NetCDF
  [rms first :1-13]
    RMSD: (:1-13), reference is first frame (:1-13).
	Best-fit RMSD will be calculated, coords will be rotated and translated.
  [center :1-13 mass origin]
    CENTER: Centering coordinates using center of mass of atoms in mask (:1-13) to
	coordinate origin.
  [volmap output/dummy.dat 0.5 0.5 0.5 :WAT@O centermask !:1-13 radscale 1.36 size 20,20,20]
    VOLMAP: Grid spacing will be 0.50x0.50x0.50 Angstroms
	Grid centered at origin.
	Gridding atoms in mask ':WAT@O'
	Dividing radii by 1.360000
	Density will wrtten to 'output/dummy.dat'
	Grid dataset name is 'VOLMAP_00002'
	Parallelizing calculation with 4 threads.
---------- RUN BEGIN -------------------------------------------------

PARAMETER FILES (1 total):
 0: tz2.ortho.parm7, 5293 atoms, 1704 res, box: Orthogonal, 1692 mol, 1691 solvent

INPUT TRAJECTORIES (1 total):
 0: 'tz2.ortho.nc' is a NetCDF AMBER trajectory, Parm tz2.ortho.parm7 (Orthogonal box) (reading 1 of 10)
  Coordinate processing will occur on 1 frames.

BEGIN TRAJECTORY PROCESSING:
.....................................................
ACTION SETUP FOR PARM 'tz2.ortho.parm7' (3 actions):
  0: [rms first :1-13]
	Target mask: [:1-13](220)
	Reference mask: [:1-13](220)
Warning: Coordinates are being rotated and box coordinates are present.
Warning: Unit cell vectors are NOT rotated; imaging will not be possible
Warning:  after the RMS-fit is performed.
  1: [center :1-13 mass origin]
	Mask [:1-13] corresponds to 220 atoms.
  2: [volmap output/dummy.dat 0.5 0.5 0.5 :WAT@O centermask !:1-13 radscale 1.36 size 20,20,20]
	Volmap: Grid mask [:WAT@O] selects 1691 atoms.
----- tz2.ortho.nc (1-1, 1) -----
Segmentation fault (core dumped)
```

**Comment by hainm** [comment]
oh, serial version is fine.

**Comment by drroe** [comment]
Thanks for the catch. Addressed by #425.
